middle_end/flambda/types/env/typing_env.rec.ml

```diff
@@ -528,16 +537,7 @@ let create ~resolver ~get_imported_names =
     next_binding_time = Binding_time.earliest_var;
     defined_symbols = Symbol.Set.empty;
     code_age_relation = Code_age_relation.empty;
-    closure_env = Some {
-      resolver;
-      get_imported_names;
-      prev_levels = Scope.Map.empty;
-      current_level = One_level.create_empty Scope.initial;
-      next_binding_time = Binding_time.earliest_var;
-      defined_symbols = Symbol.Set.empty;
-      code_age_relation = Code_age_relation.empty;
-      closure_env = None;
-    };
+    min_binding_time = Binding_time.before_earliest_var;
```

Comment:
Following on from my comment above, this should say `Binding_time.earliest_var`, no?  (Which means `before_earliest_var` can be removed from the interface.)